exchange/docs-conceptual/v1-module-mfa-connect-to-exo-powershell.md

```diff
@@ -18,7 +18,20 @@ description: "Admins can learn how to use the older Exchange Online Remote Power
 # V1 module - Connect to Exchange Online PowerShell using MFA
 
 > [!NOTE]
-> The older Exchange Online Remote PowerShell Module that's described in this article will eventually be retired. The Exchange Online PowerShell V2 module (EXO V2 module) supports MFA, so we suggest using it instead. For instructions, see [Connect to Exchange Online PowerShell](connect-to-exchange-online-powershell.md).
+**> Deprecation of MFA Module:**
+We will be removing the ability to use the “Exchange Online Remote PowerShell with MFA module” described in this article, as it is outdated and will be deprecated on 31 Dec 2022. 
+ 
+The support for MFA will end on 31st August 2022. We have ended the support so that we can focus our investment on supporting newer technologies and modern experiences. Microsoft recommends that you move to the latest GA version of V2 Module (v2.0.5) to avoid a situation where you need service or support that is no longer available.
```

Comment:
```suggestion
Support for MFA will end on August 31, 2022, so that we can focus our efforts on supporting newer technologies and modern experiences. We recommend that you move to the latest General Availability (GA) version of Exchange Online PowerShell V2 module to avoid a situation where you need service or support that is no longer available.
```